src/app/track/track.component.html

```diff
@@ -1,6 +1,28 @@
-
 <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch,requestAnimationFrame,Element.prototype.classList,URL"></script>
 <style>
+  .map-container {
```

Comment:
As there is now a .css file added anyway I'd prefer the styles to go into the new css file.

Reply:
Oh i forgot to delete the css file again. It was not working. No idea why, I tried several thing and then falled back to the html style variant. Will remove the css file again. Maybe someone who is more familar with angular can take a look

Reply:
Update: 
Before (broken): `styleUrls: ['./track.component.css']
`
After (working): 
`    styleUrls: ['track.component.css']
`